buildbot/configure.py

```diff
@@ -24,10 +24,12 @@ def do_configure(args):
                  "-DLLVM_EXTERNAL_PROJECTS=sycl;llvm-spirv",
                  "-DLLVM_EXTERNAL_SYCL_SOURCE_DIR={}".format(sycl_dir),
                  "-DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR={}".format(spirv_dir),
-                 "-DLLVM_ENABLE_PROJECTS=clang;llvm-spirv;sycl",
+                 "-DLLVM_ENABLE_PROJECTS=clang;sycl;llvm-spirv",
```

Comment:
Aligns the order with the order of `-DLLVM_EXTERNAL_PROJECTS`.

Reply:
Thanks for explanation! I was going to ask why do you need this change at all

Reply:
You can then add this explanation as a comment. :-)

Reply:
This is more a cleanup - not sure a comment in code would add much useful info?